File: C++/WarunekWTablicy.cpp
#include <vector>
#include <iostream>

int main() 
{
    int n, x;
    char condition;
    std::cin >> n;
    std::vector<int> numbers(n);
    for (int i = 0; i < n; i++)
        std::cin >> numbers[i];
    std::cin >> condition >> x;
    if (condition == '>')
    {
        for (int i = 0; i < n; i++)
            if (numbers[i] > x)
                std::cout << numbers[i] << '\n';
    }
    else if (condition == '<')
    {
        for (int i = 0; i < n; i++)
            if (numbers[i] < x)
                std::cout << numbers[i] << '\n';
    }
    return 0;
}

File: Python/Warunek_w_tablicy.py
n = int(input())
numbers = [int(input()) for _ in range(n)]
condition, main_number = input().split()
main_number = int(main_number)
if condition == '>':
    output = [number for number in numbers if number > main_number]
else:
    output = [number for number in numbers if number < main_number]
for number in output:
    print(number)
